atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/SchemaMutationLock.java

```diff
@@ -253,31 +276,62 @@ private void waitForSchemaMutationLockWithoutCas() throws TimeoutException {
     }
 
     private void schemaMutationUnlock(long perOperationNodeId) {
+        boolean unlockDone = false;
+        boolean isInterrupted = false;
+        long currentSleepMillis = TIME_BETWEEN_LOCK_ATTEMPT_ROUNDS_MILLIS;
+        for (int unlockRetryCount = 0; unlockRetryCount < MAX_UNLOCK_RETRY_COUNT; unlockRetryCount++) {
+            unlockDone = trySchemaMutationUnlockOnce(perOperationNodeId);
+            if (unlockDone) {
+                break;
+            }
+
+            currentSleepMillis = Math.min(2 * currentSleepMillis, TIME_BETWEEN_LOCK_ATTEMPT_ROUNDS_MILLIS_CAP);
```

Comment:
This duplicates the Math.min(...) logic from line 209 above; consider extracting a method.